algebra/src/curves/bn_382/g.rs

```diff
@@ -0,0 +1,91 @@
+use crate::{
```

Comment:
I'd opt to put the dual BN-382 in a separate folder. It is a curve on its own, and the fact that it forms a cycle with the BN does not justify the same folder (we don't do that with the MNT either).